projects/samples/robotbenchmark/pit_escape/worlds/pit_escape.wbt

```diff
@@ -13,25 +13,29 @@ Viewpoint {
 TexturedBackground {
```

Comment:
We could improve the viewpoint location and orientation to face the sun, the rendering is much nicer.